REQUIRE

```diff
@@ -1,5 +1,4 @@
 julia 0.4.6
-JSON 0.8.0
+JSON 0.6.0
 MathProgBase 0.5.4
 JuMP 0.14.0
-Compat 0.9.3
```

Comment:
As far as I can tell, this was never a req in the first place.  All test pass without including it.